config/data/experiments.yml

```diff
@@ -0,0 +1,22 @@
+---
+- id: 123
+  active: true
```

Comment:
I'm also considering adding an experiment end date, which will make it easier to ensure that the experiment only runs for 30 days. When appointments are being scheduled, in addition to checking to see if the patient's experiment feature flag is on, it can also check to confirm that the experiment is still running.